packages/noya-designsystem/src/utils/getGradientBackground.tsx

```diff
@@ -0,0 +1,19 @@
+import Sketch from '@sketch-hq/sketch-file-format-ts';
+import { sketchColorToRgbaString } from '..';
+
+export function getGradientBackground(
+  value: Sketch.GradientStop[],
+  type?: Sketch.GradientType,
+) {
+  const color = [...value]
+    .sort((a, b) => a.position - b.position)
+    .map((g) => `${sketchColorToRgbaString(g.color)} ${g.position * 100}%`)
+    .join(', \n');
+
+  const position =
+    type === Sketch.GradientType.Radial
+      ? 'radial-gradient(circle'
+      : `linear-gradient(${type !== undefined ? 0 : 90}deg`;
```

Comment:
Why does an `undefined` type change the direction?

Reply:
I use this component for showing the preview in the button too, in theory it would get the degree from the gradient to show the direction correctly. If it's undefined it means that we are just showing the gradient bar

Reply:
Ah ok. I think it'd be clearer to pass the direction explicitly. E.g. `getGradientBackground(values, Sketch.GradientType.Linear, 'horizontal') or something